### ISSUE TITLE
Accessibility.

### ISSUE THREAD

**Issue by AsmahanM** [AsmahanM/Pomodoro-Timer#12]
- [ ] Conduct an lighthouse test.
- [ ] Refer to the WCAG's accessibility checklist.
- [ ] Users should be able to use and navigate the app with only their keyboards.
- [ ] Users should be able to use and navigate the app with a screen reader.
